tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -0,0 +1,37 @@
+*** Settings ***
+Resource            ../../../Resources/ODS.robot
+Resource            ../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+
+Suite Setup         Set Library Search Order  SeleniumLibrary
+Suite Teardown      End Web Test
+
+
+*** Test Cases ***
+Verify Any OpenShift User Can Spawn JupyterLab Notebooks
+    [Documentation]    Verify that any OpenShift user should also be a RHODS user
+    [Tags]    Sanity
+    ...       Tier1
+    ...       ODS-1164
+    Verify Groups ConfigMap Contains The Expected OCP Groups
+    Verify User Can Spawn A Notebook
+
+
+*** Keywords ***
+Verify Groups ConfigMap Contains The Expected OCP Groups
```

Comment:
The name of the keyword Verify Groups ConfigMap Contains The Expected OCP Groups might be confusing. With this name I would expect the keyword to work with any ConfigMap, but it only works with RHODS Groups Config ConfigMap, because other ConfigMaps will have a different configuration data.

Reply:
updated